hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -125,12 +130,13 @@
     private final AtomicInteger scaleUpScheduledCount = new AtomicInteger();
 
     JobCoordinationService(NodeEngineImpl nodeEngine, JetService jetService, JetConfig config,
-                           JobRepository jobRepository) {
+                           JobRepository jobRepository, ObservableRepository observableRepository) {
```

Comment:
making a note here that this change requires an enterprise PR

Reply:
https://github.com/hazelcast/hazelcast-jet-enterprise/pull/85

Reply:
Ahh, your latest changes completely invalidate this, will delete the enterprise PR.